assets.mjs

```diff
@@ -0,0 +1,42 @@
+import sharp from "npm:sharp@0.33.5";
+
+// Names of the SVGs to use for generation
+const logoFileNames = ["css", "css.square", "css.dark", "css.light"];
+
+// Start the generation process for each logo
+for (const logoFileName of logoFileNames) {
+  // Extract the type of the logo, when the type is not provided it defaults to "primary"
+  const [logoName, logoType = "primary"] = logoFileName.split(".");
+
+  // Load the SVG file into sharp
+  const image = await sharp(`${logoFileName}.svg`);
+
+  // Specify and prepare the output folder
+  const outputFolder = `./${logoType}`;
+
+  // Remove the output folder if it exists
+  try {
+    await Deno.remove(outputFolder, { recursive: true });
+  } catch (error) {
+    if (!(error instanceof Deno.errors.NotFound)) {
+      throw error;
+    }
+  }
+
+  await Deno.mkdir(outputFolder);
+
+  const formats = ["png", "webp", "avif"];
```

Comment:
What about [`jxl`](https://jpegxl.info/index.html)? It [seems like `libvips` supports it](https://www.libvips.org/), and [`sharp` too but not with the prebuilt binaries](https://sharp.pixelplumbing.com/api-output#jxl)

Reply:
Nevermind, it's [only at ~15% support so far](https://caniuse.com/jpegxl)